api/middleware.js

```diff
@@ -170,7 +170,9 @@ const parseArray = function(field, checkFunc) {
   });
 };
 
-
+const parseBool = function(value) {
+  return (value == "true") || (value == "True");
```

Comment:
maybe just lowercase the input first so you only have to check one value and it works for any case?